kratos/tests/test_processes.py

```diff
@@ -730,6 +730,7 @@ def test_point_output_process_node(self):
                             "position"         : [0.5, 0.25, 0.0],
                             "model_part_name"  : "Main",
                             "output_file_name" : "node_output",
+                            "output_folder"    : "",

```

Comment:
I think it would be good if one of these processes would actually output into a folder
then it would also be covered in a test

Reply:
Added to test_point_output_process_node